### PR TITLE
Add the Body section into the Portfolio project

### PR DESCRIPTION
**Pull request by hienphan0111**
Build the middle section of the mobile website
  - "Works" section.
  - the "About myself" section (placed after the projects and before the contact form)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -135,13 +125,219 @@ h1 {
...
+}
+
+.project {
+  background-color: #fff;
+  border-radius: 20px;
+  box-shadow: 0 3px 4px 0 #344563;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 20px;
+  padding: 25px;
```
Comment (Debas-31) [inline styles.css]: - [x] Great job in implementing your work section design :+1: , but you have to work on small changes to stick to the template you choose. The alignment of working section not properly align. Kindly fix the issue :wrench: 

***Template design***
Screenshot from 2022-12-02 14-10-41



***Your design***

Screenshot from 2022-12-02 14-18-47

File: index.html [hunk 2]
```diff
@@ -29,12 +35,133 @@ <h1 class="main-header">I'm Hien <br> Glad to see you!</h1>
...
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="langue">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button class="read-more"><a href="http://"> See Project</a></button>
+                </div>
+            </div>
```
Comment (Debas-31) [inline index.html]: - [x] I think adding the button effects is better:mag: . The button (e.g., "See project") is a link or button tag, and follow the button styling: Active, Hover, and Pressed; see design below.

> Screenshot from 2022-12-02 15-24-34

File: index.html [hunk 3]
```diff
@@ -29,12 +35,133 @@ <h1 class="main-header">I'm Hien <br> Glad to see you!</h1>
...
+        </section>
+        <section class="about">
+            <div class="about-info">
+                <h2>About <br> Myself</h2>  
+                <p class="about-des">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>  
+                <div class="social">
+                    <p>let's connect</p>
+                    <ul class="social-icon">
```
Comment (Debas-31) [inline index.html]: - [x] Kindly change the paragraph tag used to wrap `let's connect` to header tags from (h3 - h6) to show your design differs from the paragraph :+1:

**Review by Debas-31 — CHANGES_REQUESTED** [review]
> ## Hi @hienphan0111  :wave:
# Status: Required Change :recycle:
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 
 - Linters setup :heavy_check_mark:
- GitHub flow :heavy_check_mark: 
- PR title and summary :heavy_check_mark: 
- Commit message :heavy_check_mark:
- Template usage :heavy_check_mark: 
## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 4]
```diff
@@ -121,27 +111,241 @@ h1 {
...
+  border-radius: 20px;
+  box-shadow: 0 3px 4px 0 #344563;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 24px 0;
+  padding: 24px;
+  width: 100%;
+}
+
```
Comment (mukhammadrizooff) [inline styles.css]: - [ ] [REQUIRED] Good job so far 👍 Please fulfill the request made by the [previous reviewer](https://github.com/hienphan0111/Hien-Portfolio/pull/3#discussion_r1038080774), according to which you still have `no space` on the `right` side of the `work` section elements and in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=hdw5gHzcXazz2Jky-0) every work element is `centered` and given the `same` space value from all sides, please fulfill this request 🙏🙏🙏

image

File: index.html [hunk 5]
```diff
@@ -27,14 +33,132 @@
...
+                    </ul>
+                    <p class="project-des">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="langue">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button type="button" class="btn read-more"><a href="http://"> See Project</a></button>
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far 👍 Please fulfill the request made by the [previous reviewer](), please implmen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=hdw5gHzcXazz2Jky-0) `button` effects🙏🙏🙏

### Your design ❌ 
image

### Figma design ✔️

image

File: index.html [hunk 6]
```diff
@@ -27,14 +33,132 @@
...
+                </div>
+            </div>
+        </section>
+        <section class="about">
+            <div class="about-info">
+                <h2>About <br> Myself</h2>  
+                <p class="about-des">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>  
+                <div class="social">
+                    <h3>let's connect</h3>
+                    <ul class="social-icon">
```
Comment (mukhammadrizooff) [inline index.html:141]: - [ ] [OPTIONAL] Good job so cool 💯 👍

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @hienphan0111, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Yerimah — APPROVED** [review]
# STATUS: APPROVED 🥇 🔥 
Hi @hienphan0111 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

68747470733a2f2f36342e6d656469612e74756d626c722e636f6d2f38376530623634363963393537626638393663646563316361623634366163662f74756d626c725f696e6c696e655f706b316264616b434370317270637239615f3530302e676966

## Highlights
- No Linter error ✔️
- GitHub flow ✔️
- Descriptive PR title and summary ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**